src/main/scala/it/pagopa/interop/apigateway/service/impl/PartyRegistryProxyServiceImpl.scala

```diff
@@ -0,0 +1,50 @@
+package it.pagopa.interop.apigateway.service.impl
+
+import com.typesafe.scalalogging.{Logger, LoggerTakingImplicit}
+import it.pagopa.interop.apigateway.service.{PartyRegistryInvoker, PartyRegistryProxyService}
+import it.pagopa.interop.commons.utils.extractHeaders
+import it.pagopa.interop.partyregistryproxy.client.api.InstitutionApi
+import it.pagopa.interop.partyregistryproxy.client.invoker.{ApiError, BearerToken}
+import it.pagopa.interop.partyregistryproxy.client.model._
+import it.pagopa.interop.commons.utils.TypeConversions._
+import it.pagopa.interop.commons.logging.{CanLogContextFields, ContextFieldsToLog}
+import it.pagopa.interop.commons.utils.errors.GenericComponentErrors
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class PartyRegistryProxyServiceImpl(invoker: PartyRegistryInvoker, api: InstitutionApi)(implicit ec: ExecutionContext)
+    extends PartyRegistryProxyService {
+  implicit val logger: LoggerTakingImplicit[ContextFieldsToLog] =
+    Logger.takingImplicit[ContextFieldsToLog](this.getClass)
+
+  override def getInstitutionByExternalId(origin: String, originId: String)(implicit
+    contexts: Seq[(String, String)]
+  ): Future[Institution] = for {
+    (bearerToken, correlationId, ip) <- extractHeaders(contexts).toFuture
+    request = api.getInstitutionByExternalId(
+      // xCorrelationId = correlationId,
```

Comment:
🧹

Reply:
Nope, once [this](https://github.com/pagopa/interop-be-party-registry-proxy/pull/144) gets merged you need to add these params